library/mr_provisioner_preseed.py

```diff
@@ -1,12 +1,12 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
```

Comment:
I don't think we're quite ready to require python3. If you'd like to propose this, please do so in another PR and apply the change to every module consistently.